src/main/java/pickleib/utilities/email/EmailInbox.java

```diff
@@ -28,6 +29,7 @@ public class EmailInbox {
     String secureCon;
     String email;
     String emailApplicationPassword;
+    static long emailAcquiringTimeout = Long.parseLong(ContextStore.get("email-acquiring-timeout", "45000"));
```

Comment:
Why have the same variable declared twice?

Reply:
The first variable is declared in EmailAcquisition.class, the second one is in EmailInbox.class.